src/bioregistry/data/bioregistry.json

```diff
@@ -15937,7 +15937,7 @@
     "homepage": "https://www.chictr.org.cn",
     "logo": "https://www.chictr.org.cn/images/top.jpg",
     "name": "Chinese Clinical Trial Registry",
-    "pattern": "^ChiCTR\\d+$",
+    "pattern": "^ChiCTR\\d+$|^ChiCTR-[A-Z]+-\\d+$",
```

Comment:
This could be simplified by keeping just ^ChiCTR and \\d+$ and adding the -[A-Z]+- as optional in the middle.